src/layout/Header.jsx

```diff
@@ -209,8 +209,12 @@ function MobileCalculatorButton() {
 function LoginButton() {
   const countryId = useCountryId();
   const { loginWithRedirect, isAuthenticated, user, isLoading } = useAuth0();
+  const displayCategory = useDisplayCategory();
+
   const desiredHeight =
-    style.spacing.HEADER_HEIGHT - BAR_TOP_PADDING - BAR_BOTTOM_PADDING;
+    displayCategory === "mobile"
+      ? 44
+      : style.spacing.HEADER_HEIGHT - BAR_TOP_PADDING - BAR_BOTTOM_PADDING + 4;
```

Comment:
One thing we try to avoid as much as possible is avoiding "magic numbers," numbers where we don't really know what they mean, and that's the case with both the 44 and the 4 here. Instead of this, from a code quality perspective, could you change BAR_TOP_PADDING and BAR_BOTTOM_PADDING (defined around line 25) to be 10 when on mobile and 8 on tablet and desktop, then keep this line like it was before? This applies to the change below around line 329, as well.